docs/en/matter/ep_temperature_controlled_cabinet.rst

```diff
@@ -0,0 +1,295 @@
+##################################
+MatterTemperatureControlledCabinet
+##################################
+
+About
+-----
+
+The ``MatterTemperatureControlledCabinet`` class provides a temperature controlled cabinet endpoint for Matter networks. This endpoint implements the Matter temperature control standard for managing temperature setpoints with min/max limits, step control (always enabled), or temperature level support.
+
+**Important:** The ``temperature_number`` and ``temperature_level`` features are **mutually exclusive**. Only one can be enabled at a time. Use ``begin(tempSetpoint, minTemp, maxTemp, step)`` for temperature_number mode or ``begin(supportedLevels, levelCount, selectedLevel)`` for temperature_level mode.
```

Comment:
Move this part after explaining the `Temperature Number Mode` and `Temperature Level Mode` in the `Features` section.

Reply:
done;